helm/YataiService/templates/postgres-service.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.postgres.enabled -}}
+apiVersion: v1
+kind: Service
+metadata:
+  name: postgres
```

Comment:
missing namespace

Reply:
from this discussion, it seems like the standard is to specify the namespace in the `kubectl` step rather than in the helm chart itself https://github.com/helm/helm/issues/3553#issuecomment-521424132

Reply:
wow, there are a lot of discussions on that issue, and it seems community and the helm teams have different ideas in mind.

Maybe we can change the service/deployment name for postgres to something like `bentoml-postgres'? A lot of companies are using postgres in their K8s cluster. I don't want any accidents to happen when they deploy or remove Yatai service from their cluster.